### ISSUE TITLE
Add show method to io.Stack object

### ISSUE THREAD

**Issue by dganguli** [spacetx/starfish#67]
Right now, the API for displaying an image stack looks like:

```
from starfish.io import Stack
s = Stack()
s.read(in_json)
tile(s.squeeze());
```

It would be nice if the last line in the above snippet looked like:

`s.show()`

**Comment by dganguli** [comment]
s.show() should label all axes (hybs, chs) and automatically max project across z for 3D data.

**Comment by ambrosejcarr** [comment]
Added related method in #232 .

**Comment by ambrosejcarr** [comment]
@dganguli do you think there's benefit in implementing this, given what we currently have implemented?

**Comment by dganguli** [comment]
nope. this was addressed in a much better manner by #217 and is actively being improved and supported.
